Sources/AutobahnDescription/AutobahnDescription.swift

```diff
@@ -111,26 +117,35 @@ public final class Autobahn<H: Highway> {
         } catch {
             self.onErrorHandler?(secondArg ?? "", error)
         }
+        highwayEnd = Date()
```

Comment:
We should print at the end how long the drive took. (And later how much time was saved :trollface:)

Reply:
✅